### PR TITLE
Balance notifications

### PR DESCRIPTION
**Pull request by 0xniksh**

### CODE REVIEW

File: app/tasks.py [hunk 1]
```diff
@@ -62,3 +70,58 @@ def process_history_staking_event():
...
+@app.task(max_retries=3, default_retry_delay=10)
+def monitor_onramp_bridge_balance():
+    logger.info("Monitoring onramp bridge balance")
+    redis = get_redis()
+    chain_id = chains.blast_sepolia.id if settings.app_env == "dev" else chains.blast.id
+
+    async def get_and_log_balance():
+        try:
+            web3 = await web3_node.get_web3(chain_id=chain_id)
+            balance_wei, balance_in_cache, blast_price = await asyncio.gather(
```
Comment (qringie) [inline app/tasks.py:84]: Why would there be a balance from the cache? It seems that it is necessary to always get an up-to-date balance without delays

**Review by qringie — CHANGES_REQUESTED** [review]

File: app/tasks.py (hunk 1 above)
Comment (0xniksh) [reply app/tasks.py:84]: It's just used to compare balance with previous execution. so there won't be spam messages if balance less than threshold.
@qringie

File: app/tg.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        )
+        await self.send(msg)
+
+    async def new_onramp_order(self, order_id: str, str_amount: str, currency: str) -> None:
+        msg = (
+            f"<b>🤑 NEW Munzen order 🤑</b>\n\n"
+            f"<b>ID:</b> {order_id}\n"
+            f"<b>Amount:</b> {str_amount} {currency}"
+        )
+        await self.send(msg)
```
Comment (astroangrycat) [inline app/tg.py:34]: > 🤑  Munzen Order
ID: ...
Amount: $100 (0.01 ETH)
User's wallet: ...

> Onramp wallet balance: 0.01 ETH (put here balance after sending crypto)

File: app/tg.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        self, blast_balance: float, usd_balance: float
+    ) -> None:
+        blast_scan_url = f"https://blastscan.io/address/{settings.onramp_sender_addr}"
+        msg = (
+            f"<b>💸 WARNING 💸</b>\n\n"
+            f"Onramp bridge balance is low\n"
+            f"{blast_balance:.6f} BLAST (<b>${usd_balance:.2f}</b>)\n"
+            f"<a href='{blast_scan_url}'>Contract</a>"
+        )
+        await self.send(msg)
```
Comment (astroangrycat) [inline app/tg.py:26]: > ⚠️ Onramp Bridge **Warning**
Onramp Bridge wallet balance is low
Balance: **0.01 ETH**
Wallet Address: `0x....` (<a href='{blast_scan_url}'>Blastscan</a>)

File: app/tg.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+    async def completed_onramp_order(self, order_id: str, tx_hash: str) -> None:
+        scanner_url = f"https://blastscan.io/tx/{tx_hash}"
+        msg = (
+            f"<b>✅ Completed Munzen order ✅</b>\n\n"
+            f"<b>ID:</b> {order_id}\n"
+            f"<a href='{scanner_url}'>Transaction</a>"
+        )
+        await self.send(msg)
```
Comment (astroangrycat) [inline app/tg.py:43]: > ✅  COMPLETED Munzen Order
ID: `...`
Transaction Munzen->Bridge: <a href="">tx_hash</a>
Transaction Bridge->User: <a href="">tx_hash</a>
Onramp wallet balance: 0.01 ETH (put here balance after sending crypto)

**Review by tripelkarmeliet — APPROVED** [review]

**Pull request merged**